test/__tests__/data/braze_router.json

```diff
@@ -841,9 +841,7 @@
         },
         {
           "error": "[Braze Deduplication]: Duplicate user detected, the user is dropped",
-          "statTags": {
```

Comment:
we had test case too to validate the previous change then !

Reply:
Yes 🙁